lib/src/actions.dart

```diff
@@ -69,14 +71,24 @@ class CustomSlidableAction extends StatelessWidget {
   /// Typically the action's icon or label.
   final Widget child;
 
+  /// {@template slidable.actions.shape}
+  /// [OutlinedBorder] of action, if is null, then [RoundedRectangleBorder]
+  /// used by default
+  /// {@endtemplate}
+  final OutlinedBorder shape;
+
+  /// {@template slidable.actions.side}
+  /// [BorderSide] of action, if is null, then [BorderSide.none] used by default
+  /// {@endtemplate}
+  final BorderSide side;
```

Comment:
We should make `BorderSide` nullable too.

Reply:
fixed